depends/libhdfs3/rpms/libhdfs3.spec

```diff
@@ -45,20 +45,6 @@ It gets rid of the drawbacks of JNI, and it has a lightweight,
 small memory footprint code base. In addition, it is easy to use and deploy.
 .
 
-%package devel
```

Comment:
I'm not sure, but we might run into problems because we will be missing all the `-devel` requirements?